docker/bench-all.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# Copyright (c) 2022, NVIDIA CORPORATION.
+
+set -e
+
+function logger {
+    echo -e "\n$@\n"
+}
+
+# Requires conda installed at /opt/conda and the ucx environment setup
+# See UCXPy-CUDA.dockerfile
+source /opt/conda/etc/profile.d/conda.sh
+conda activate ucx
+
+# Expect ucx-py checkout here
```

Comment:
Not sure what this comment means, is it a leftover or intentional?

Reply:
I think we can remove that, "ucx-py checkout" gives me the impression there should be some `git checkout` involved.

Reply:
Done.